### DISCUSSION TITLE
RAG on Azure SQL Server: Session Resources

### DISCUSSION THREAD

**Discussion by multispark** [microsoft/RAG_Hack#53] [Announcements]
This is the resources and discussion thread for this live stream: https://aka.ms/raghack/azuresql
It will be updated with links after the session.

Recording: https://www.youtube.com/watch?v=B47QNc7xRbg
Slides: [PPT](https://github.com/microsoft/RAG_Hack/blob/main/slides/RAG%20With%20Azure%20SQL.pptx)
Code sample: https://github.com/Azure-Samples/azure-sql-db-rag-langchain-chainlit

Please ask any follow-up questions here, and the speakers will see them.

**Comment by ralphke** [comment]
Hi @pamelafox, @multispark 
The PowerPoint deck for the session RAG With Azure SQL.pptx is Microsoft rights protected, as well as RAGHackResponsibleAI.pptx .
This way it cannot be opened from people external to Microsoft without an explicit grant, wich I cannot see as intention for this content.
Please remove the Microsoft Rights Management tags from the files and republish it on GitHub in this repro.
Thanks
Ralph

**Reply by JasonHaley** [reply]
I'm having the same promblem.

**Reply by pamelafox** [reply]
Sorry about that! I thought I'd changed the label, but I've just given it another go, can you try loading it again?

**Reply by multispark** [reply]
Thanks for letting us know, the Responsible AI pptx should be updated now as well.

**Reply by JasonHaley** [reply]
Yes works fine for me now. Thanks!
